src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -2058,10 +2058,10 @@ _Default:
             CheckPosition(position)
 
             If symbol Is Nothing Then
-                Throw New ArgumentNullException("symbol")
+                Throw New ArgumentNullException(NameOf(symbol))
             End If
 
-            Dim vbsymbol = symbol.EnsureVbSymbolOrNothing(Of symbol)("symbol")
+            Dim vbsymbol = symbol.EnsureVbSymbolOrNothing(Of symbol)(NameOf(position))
```

Comment:
@AdamSpeight2008 @gafter This appears to have been accidentally changed from symbol to position.